spec/helpers/value_helper_spec.rb

```diff
@@ -31,4 +31,25 @@
       expect(helper.links_to_collections(args)).to have_css 'br'
     end
   end
+
+  describe 'originInfo_date_created_tesim' do
+    it 'does not modify a year string' do
+      field = 'originInfo_date_created_tesim'
+      value = ['1966', 'anything_after_first_element_ignored']
+      document = SolrDocument.new({ field => value })
+      args = { document: document, field: field }
+      expect(helper.value_for_originInfo_date_created_tesim(args))
+        .to eq(value.first)
+    end
+    it 'does normalize a time stamp string' do
+      now_utc = Time.now.utc.to_s
+      now_loc = Time.parse(now_utc).localtime.strftime '%Y.%m.%d %H:%M%p'
```

Comment:
`#in_time_zone` is an ActiveSupport thing, I believe. Try within the context of the rails console.

```sh
$ rails c
[1] pry(main)> Time.now.in_time_zone
=> Thu, 10 Dec 2015 16:46:40 UTC +00:00
```